src/components/Pages/Pages.module.scss

```diff
@@ -32,24 +39,127 @@
   font-weight: 500;
   font-size: 14px;
   line-height: 24px;
+
+  &Label {
+    margin-bottom: 0;
+    font-size: 16px;
+  }
 }
 
 .pageLink {
   color: #033e8c;
   text-decoration: none;
 }
 
-// Contact page specific
 .pageContactSection {
-  margin-bottom: 20px;
+  // Existing styles for input
+  input[type='text'],
+  input[type='email'] {
+    width: min(80%, 400px);
+    height: 55px;
+    border: 1px solid $color-input-border;
+    border-radius: $border-radius;
+  }
+
+  // Additional styles for textarea
+  textarea {
+    width: min(100%, 800px);
+    height: 100px;
+    border: 1px solid $color-input-border;
+    border-radius: $border-radius;
+    resize: vertical;
+  }
 }
 
 .pageContactLabel {
   margin: 0;
   margin-bottom: 5px;
-  color: #2d3748;
-  font-family: 'Inter', sans-serif;
+  color: $color-label;
+  font-family: $font-family-inter;
   font-weight: 600;
   font-size: 17px;
   line-height: 20px;
+
+  span {
+    color: $color-asterisk;
+  }
+}
+.pageContactSection-grid {
+  width: fit-content;
+  display: grid;
+  place-items: center;
+  grid-template-columns: 20px 1fr;
+  gap: 8px;
+}
+.pageContactCheckbox {
+  width: 18px;
+  height: 18px;
+}
+
+.submitButton {
+  width: min(80%, 400px);
+  margin-bottom: 20px;
+  padding: 7px 0;
+  border-radius: 8px;
+  background-color: $button-bg-color;
+
+  p {
+    margin-bottom: 0;
+    color: #ffffff;
+    font-family: $font-family-inter;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 23px;
+  }
+}
+.divContainer {
+  height: calc(
+    (100vh - 76px - 32px - 25px - 25px - 92px - 70px - 502px - 50px)
```

Comment:
Can we map each of the values here as a variable? Can we add a comment explaining the values that construct this calculation? This will make it clear why these values are used.